src/dbnode/storage/shard.go

```diff
@@ -2999,14 +3009,18 @@ func (s *dbShard) logFlushResult(r dbShardFlushResult) {
 	)
 }
 
-func (s *dbShard) finishWriting(startTime time.Time, nextVersion int) error {
+func (s *dbShard) finishWriting(
+	startTime time.Time,
```

Comment:
nit: We use `blockStart` in places where `time.Time` represents the start of a block. 

Although reading further up, it looks like we use `Start, End` as options to `AggregateTilesOptions` I guess it would be clearer to have it be `BlockStart, BlockEnd` if that is what it represents?

Reply:
Renamed `startTime` to `blockStart`.